crates/ark/src/modules/positron/graphics.R

```diff
@@ -222,6 +193,142 @@ with_graphics_device <- function(
     expr
 }
 
+use_ragg <- local({
+    # Only check global option once per session
+    USE_RAGG <- NULL
+
+    function() {
+        if (is.null(USE_RAGG)) {
+            USE_RAGG <<- init_use_ragg()
+        }
+
+        USE_RAGG
+    }
+})
+
+init_use_ragg <- function() {
+    option <- getOption("ark.ragg", default = TRUE)
+
+    if (!isTRUE(option)) {
+        # Bail on any non-`TRUE` option value
+        return(FALSE)
+    }
+
+    if (!.ps.is_installed("ragg", minimum_version = "1.4.0")) {
+        # Need support for `agg_record()`
+        return(FALSE)
+    }
+
+    TRUE
+}
```

Comment:
Open to ideas if there are strong feelings against this approach

Reply:
don't see anything contentious there